public/help/help.md

```diff
@@ -30,3 +31,91 @@ The following procedure describes adding and updating settings of custom tracks.
+
+###### Query String Encoding
+The link has a [query string](https://en.wikipedia.org/wiki/Query_string#Structure) that consists of key-value pairs separated by ampersands, where the keys are not URL-econded and the values are URL-encoded. 
+
+Keys can use brackets to encode hierarchical structures such as arrays and objects. 
+   * For arrays, put a number in brackets to assign a new value to or access a value in that entry in the array.
+   * For objects, put the key's name in brackets to assign a new value to or access a value in that entry in the object. 
+
+The key's name should **not** be in quotes.
+
+###### Fields that can be included in the link URL:
+1. `tracks` 
+
+   Information about tracks. Tracks are objects consisting of `trackFile`, `trackType`, and `trackColorSettings`.
+
+   A track object in JSON format might look like this:
+   ```
+   {
+      "trackFile": "exampleData/internal/snp1kg-BRCA1.vg.xg",
+      "trackType": "graph",
+      "trackColorSettings": {
+         "mainPalette": "greys",
+         "auxPalette": "ygreys",
+         "colorReadsByMappingQuality": false
+      }
+   }
+   ```
+
+   The collection of tracks wil be encoded as query string parameters starting with tracks.
+
+   * The `trackFile` is the path (from the server working directory) or URL (any HTTP/HTTPS URL) to the track file. 
+     * Examples:
+         * File in server working directory: `tracks[0][trackFile]=exampleData/internal/snp1kg-BRCA1.vg.xg`.
+         * HTTP/HTTPS URL: `tracks[0][trackFile]=https://public.gi.ucsc.edu/~anovak/graphs/trivial-brca1.vg`.
+   * The `trackType` specifies the type of the track, which can be `graph`, `haplotype`, `read`.
+      * Example: `tracks[0][trackType]=graph`.
+   * `trackColorSettings` (optional): the color information for the track. It has subfields:
+      * `mainPalette` and `auxPalette` (optional): The "main" and "auxilliary" color palettes to use, respectively. The values for each palette can be a hex code starting with a `#`, or one of the premade palette names: `greys`, `ygreys`, `blues`, `reds`, `plainColors`, or `lightColors`.
+      * The palettes are used differently for different track types.
+         * Graphs: the `mainPalette` colors the primary reference path while `auxPalette` is used for the other paths. 
+         * Haplotypes: only the `mainPalette` is used. 
+         * Reads: the `mainPalette` colors the forward-strand reads and the `auxPalette` colors the reverse-strand reads. 
+         * Examples:
+            * `tracks[0][trackColorSettings][mainPalette]=greys`.
+            * `tracks[0][trackColorSettings][auxPalette]=ygreys`.
+      * `colorReadsByMappingQuality` (optional) is a boolean value that determines if reads are colored based on their mapping quality or not. It defaults to `false`. 
+         * Example: `tracks[0][trackColorSettings][colorReadsByMappingQuality]=true`.
+   
+
+2. `region`
+
+   Region coordinates. This is documented at step 3 of
```

Comment:
"This" here sounds a little vague; we should maybe say that the syntax for specifying genomic coordinates is what is documented there.